src/app/pages/FeedPage/components/Twit.tsx

```diff
@@ -70,15 +70,15 @@ export default function Twit({
         </div>
         <div className={styles.twitBox__article}>{content}</div>
         <div className={styles.twitBox__buttons}>
-          <div className={styles.buttons__box + 'hover:text-[#f91880]'}>
+          <div className={styles.buttons__box + 'hover:text-pink'}>
             <SvgLike />
             <span className={styles.box__text}>{likes}</span>
           </div>
-          <div className={styles.buttons__box + 'hover:text-[#1d9bf0]'}>
+          <div className={styles.buttons__box + 'hover:text-logo-light'}>
```

Comment:
I very often see `logo-light` are used without `logo-dark`. Maybe just copy `logo-light` to separate color? And if possible use that color in `logo-light` and by that I don't mean copy and paste same color but you make it more programmatically or just do something like
```typescript
{
  newColor: 'some-color-here' ,
  logo: {
    light: 'newColor',
    dark: 'same-color-value-as-before',
}
```